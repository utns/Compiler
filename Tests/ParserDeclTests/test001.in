const
    b = 1.0;