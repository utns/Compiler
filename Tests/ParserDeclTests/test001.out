integer                  type 
float                    type 
char                     type 
b                 const float               1
